### ISSUE TITLE
Halo infinite image freezes midmatch but the game is still running. [SteamDeck]

### ISSUE THREAD

**Issue by DanielCardiel** [ValveSoftware/Proton#6441]
# Compatibility Report
- Name of the game with compatibility issues: Halo Infinite
- Steam AppID of the game: 1240440

## System Information

Steam Deck, updated to version 3.4
- GPU: 
- Driver/LLVM version: 
- Kernel version: 
- https://gist.github.com/DanielCardiel/fbc0ca78b468ba66cb26f96cde99b85f
- Proton version: experimental-7.0-20230104

## I confirm:
- [x ] that I haven't found an existing compatibility report for this game.
- [x ] that I have checked whether there are updates for my system available.

## Symptoms 
Halo infinite image freezes mid-match but the game is still running.  I mean, I can still shot move because I can hear it but the image does not update. Some times it comes back after some seconds. But from the last 10 times that happened, once came back.
This started in December, around the 10th. Sadly this has made the game unplayable for me, I already received a temp ban for being inactive or abandoning the matches...
I did reinstall the game, after verifying the files, which did not help.

## Reproduction
I have noticed that starts with specific sounds, sometimes it happens when someone shoots for the first time. One it happened when the invisibility ran out.
no specific map is needed.
the majority of the times happens in the first minutes of the games, but has happened to me that it happened in the second match.

[steam-1240440.zip](https://github.com/ValveSoftware/Proton/files/10368337/steam-1240440.zip)

**Comment by DanielCardiel** [comment]
New info.
I saw something in the DX log files. seems that the overlay could not update the FPS. not sure if this is a consequence or a reason. The game froze and came back after 1 min, but the PFS where erratic. Then the controls were weird, the buttons were not correct. This happened to me once before with XCOM2.
I deactivated the FPS overlay and did not crash, I managed to reach a 3rd match. Not sure if this is relevant tho...

Anyway, here more log files. Might be useful 
[HaloInfinite_dxgi.log](https://github.com/ValveSoftware/Proton/files/10368463/HaloInfinite_dxgi.log)
[HaloInfinite_d3d11.log](https://github.com/ValveSoftware/Proton/files/10368464/HaloInfinite_d3d11.log)

Log files too big, so: https://we.tl/t-MqEnryrHxt
